lesson02/elb_asg/userdata.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+## hostname setup
+hostnamectl set-hostname ${HOSTNAME}
+#Update installed pacakges
+ #Install docker
+yum install -y docker
+ #Start Docker
+sudo service docker start
+ #Add ec2-user to the docker group
+#All commands here are executed as super admin
+#but still, let's add the ec2-user to the docker group
+usermod -a -G docker ec2-user 
+ #Run the nginx 
+docker run -d -p 80:80 nginx:${docker_tag}
```

Comment:
sync this file with lesson01, the `--restart=always` flag is missing.

Reply:
okok, I accidently sent that file 😅